src/video_core/renderer_vulkan/vk_descriptor_pool.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <memory>
+#include <vector>
+
+#include "common/common_types.h"
+#include "video_core/renderer_vulkan/declarations.h"
+#include "video_core/renderer_vulkan/vk_descriptor_pool.h"
+#include "video_core/renderer_vulkan/vk_device.h"
+#include "video_core/renderer_vulkan/vk_resource_manager.h"
+
+namespace Vulkan {
+
+// Prefer small grow rates to avoid saturating the descriptor pool with barely used pipelines.
+static constexpr std::size_t SETS_GROW_RATE = 0x20;
```

Comment:
```suggestion
constexpr std::size_t SETS_GROW_RATE = 0x20;
```

const/constexpr file-scope variables are internally linked by default.